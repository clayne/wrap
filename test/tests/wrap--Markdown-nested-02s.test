wrap | /dev/null | -u | md-nested-02s.md | 0
