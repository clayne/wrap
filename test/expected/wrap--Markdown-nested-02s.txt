1. First item of outer list.

    + First item of nested list.

2. Second item of outer list.
